docs/breaking-changes.md

```diff
@@ -27,9 +27,20 @@ preload scripts _do_ depend on Node, either refactor them to remove Node usage
 from the renderer, or explicitly specify `sandbox: false` for the relevant
 renderers.
 
+### Removed: `skipTaskbar` on Linux
+
+See `skipTaskbar` discussion in 19.0 below. This feature is not available on
+Wayland. Since most modern Linux desktops are transitioning to Wayland, this
+feature will be removed for Linux.
+
 ## Planned Breaking API Changes (19.0)
 
-*None (yet)*
+### Unsupported: `skipTaskbar` on Linux
```

Comment:
I missed this, but all breaking changes in this document should use one of the types described at the top of the document (API Changed / Behavior Changed / Default Changed / Deprecated / Removed).

What does `Unsupported` mean?

Reply:
Followup in https://github.com/electron/electron/pull/33479